modules/service-ca-certificates.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="service-ca-certificates_{context}"]
+= Service CA certificates
+
+== Purpose
+
+`service-ca` is an Operator that creates a self-signed CA, which is valid for one year,
+when an {product-title} cluster is deployed.
+
+== Expiration
+
+A custom expiration term is not supported. The self-signed CA is stored in a
+secret with qualified name `service-ca/signing-key` in fields `tls.crt`
+(certificate(s)), `tls.key` (private key), and `ca-bundle.crt` (CA bundle).
+
+Other services can request a service serving certificate by annotating a
+service resource with `service.beta.openshift.io/serving-cert-secret-name
+<secret name>`. In response, the Operator generates a new certificate, as
+`tls.crt`, and private key, as `tls.key` to the named secret. The certificate
+is valid for two years.
+
+Other services can request that the CA bundle for the service CA be injected
+into APIService or ConfigMap resources by annotating with
+`service.beta.openshift.io/inject-cabundle: true` to support validating
+certificates generated from the service CA. In response, the Operator writes its
+current CA bundle to the `CABundle` field of APIService or as `service-ca.crt`
+to a ConfigMap.
+
+As of {product-title} 4.3, the service CA and the certificates it generates are
+not rotated automatically. This is problematic for early adopters. Without
+rotation, a service CA will expire one year after initial cluster deployment.
+
+It is possible to manually trigger rotation of the service CA by deleting the
+signing secret (qualified name: `service-ca/signing-key`). This will result in
+the creation of a new self-signed service CA, the regeneration of all serving
+certificates, and the update of all resources annotated for CA bundle injection.
+
+Manual rotation does not maintain trust with the previous CA, which might result
+in service disruption until all clients and services are refreshed to use the
+latest key material.
+
+Automated rotation will be available in a future release of {product-title}. If
+the service CA has less than six months remaining in its validity bounds, it
+will be replaced by a new CA configuration that supports trusting the previous
+CA for six months. This is intended to provide a grace period for affected
+services to refresh their key material. Forced automated rotation will also be
+supported, allowing an administrator to trigger a trust-maintaining rotation at
+any time.
+
+== Services that use it
```

Comment:
Is this list comprehensive? 
Should we denote that it might not be comprehensive.